### PR TITLE
EN_Reboot: Various fixes

### PR DESCRIPTION
**Pull request by Frey8914**
-Fix NPC names.
-Fix EQ text.
-Fix tutorials.
-Translate system Lobby Actions.
-Fix Mining Rig Defense.
-Fix Social Mag Boost announcement.
-Fix clipping.
-Fix capitalization.
-Fix Block movement.
-Fix tips.

### CODE REVIEW

File: Translated/UI/actor_name_npc.csv [hunk 1]
```diff
@@ -37,7 +37,7 @@ Npc_7609001#0,"""Maurier"""
 Npc_7610001#0,"""Waykes"""
 Npc_7611001#0,"""Nandel"""
 Npc_7612001#0,"""Hoxie"""
-Npc_7613001#0,"""Taoyuan"""
+Npc_7613001#0,"""Towen"""
```
Comment (SynthSy) [inline Translated/UI/actor_name_npc.csv]: we never agreed on changing this

File: Translated/UI/actor_name_npc.csv [hunk 2]
```diff
@@ -69,7 +69,7 @@ Npc_7663001#0,"""Sharkey"""
 Npc_7664001#0,"""Hounyan"""
 Npc_7667001#0,"""Nesca"""
 Npc_7687001#0,"""Traces"""
-Npc_7688001#0,"""Dementeluna"""
+Npc_7688001#0,"""Dimenteluna"""
```
Comment (SynthSy) [inline Translated/UI/actor_name_npc.csv:72]: is there a specific reason for this?

File: Translated/UI/actor_name_npc.csv [hunk 3]
```diff
@@ -57,7 +57,7 @@ Npc_7646001#0,"""Bill"""
 Npc_7648001#0,"""Fyne"""
 Npc_7649001#0,"""Bosey"""
 Npc_7653001#0,"""Matilda"""
-Npc_7654001#0,"""Kommuna"""
+Npc_7654001#0,"""Communa"""
```
Comment (SynthSy) [inline Translated/UI/actor_name_npc.csv:60]: is there a specific reason for this?

File: Translated/UI/actor_name_npc.csv [hunk 4]
```diff
@@ -16,7 +16,7 @@ Npc_7199001#0,"""Guiden"""
 Npc_7200001#0,"""Kartevie"""
 Npc_7201001#0,"""Somua"""
 Npc_7203001#0,"""Franme"""
-Npc_7205001#0,"""Ravonna"""
+Npc_7205001#0,"""Labona"""
```
Comment (SynthSy) [inline Translated/UI/actor_name_npc.csv]: double n please, it looks off with just one

**Review by SynthSy — CHANGES_REQUESTED** [review]

File: Translated/UI/actor_name_npc.csv (hunk 1 above)
Comment (Frey8914) [reply Translated/UI/actor_name_npc.csv]: His name like the other guards is based on cardinal directions. His is based on East, Tou, not a municipality in northwestern Taiwan that means peach garden (桃園).

File: Translated/UI/actor_name_npc.csv (hunk 2 above)
Comment (Frey8914) [reply Translated/UI/actor_name_npc.csv:72]: Dimenteluna is the senior telegrapher in the Central Tower in Central City who tells you about PSO2<->NGS dimensions, she's far from demented.

File: Translated/UI/actor_name_npc.csv (hunk 3 above)
Comment (Frey8914) [reply Translated/UI/actor_name_npc.csv:60]: Communa in the Central Tower in Central City tells you about communication functions, such as chat, group chat, autowords and stamps. She's not based on a Russian ship named Kommuna which means Commune, "a group of people living together and sharing possessions and responsibilities."

File: Translated/UI/actor_name_npc.csv (hunk 1 above)
Comment (SynthSy) [reply Translated/UI/actor_name_npc.csv]: Then it should be Touwen to properly emphasize the Tō.

File: Translated/UI/actor_name_npc.csv (hunk 3 above)
Comment (SynthSy) [reply Translated/UI/actor_name_npc.csv:60]: To be fair, i never thought it was a russian ship, just a russian name.

File: Translated/UI/actor_name_npc.csv (hunk 2 above)
Comment (SynthSy) [reply Translated/UI/actor_name_npc.csv:72]: Sounds good to me. Next time add your explanations in your PR as discord messages will be drowned out. The next time you forget to do this, we'll reject it right off.

File: Translated/UI/actor_name_npc.csv (hunk 4 above)
Comment (Frey8914) [reply Translated/UI/actor_name_npc.csv]: If it's made double it misses the pun on Laboratory to become Labonnatory.

Comment (SynthSy) [reply Translated/UI/actor_name_npc.csv]: Pun is already missed considering I wouldn't have caught this in the first place. You could argue it has lab in it and no one will have batted an eye. Add the double N.

File: Translated/UI/region_mag_common.csv [hunk 5]
```diff
@@ -1,9 +1,9 @@
-BuffToast_Prefix#0,"""'s \""Mag Boost\"": """
+BuffToast_Prefix#0,"""'s \""Social Mag Boost\"": """
```
Comment (SynthSy) [inline Translated/UI/region_mag_common.csv]: I've told you not to change this file.

File: Translated/UI/ui_tips_reb.csv [hunk 6]
```diff
@@ -25,11 +25,11 @@ Dead_r01_gd_gia_sw_002#0,"""If you focus your attacks on Daedal Sword's<br>foot,
 Dead_r01_gd_gia_ax_001#0,"""Watch Daedal Axe closely to see when it's about<br>to perform its strongest attack, then counter it."""
 Dead_r01_gd_gia_ax_002#0,"""Daedal Axe can be Downed<br>by Ice-Attribute attacks."""
 Dead_r01_gd_sol_rai_001#0,"""If Pedas Vera targets you with its punching<br>attack, try to focus on avoiding it until it stops."""
-Dead_r01_gd_sol_rai_002#0,"""Breaking the glowing crystals that appear during<br>Pedas Vera's laser attack will expose its weak point."""
+Dead_r01_gd_sol_rai_002#0,"""Breaking the crystal that glows during<br>Pedas Vera's laser attack will expose its weak point."""
 Dead_r01_nat_bea_ba_001#0,"""When Cragbear grabs you, you can escape<br>by rapidly entering movement inputs."""
 Dead_r01_nat_bea_ba_002#0,"""Cragbear can launch long-range<br>shockwaves by swinging its arms."""
 Dead_r01_nat_fan_m_001#0,"""アードバンサーは、<br>前足を開いて地面を踏みしめたあとに咆哮を放ってくる。"""
-Dead_r01_nat_fan_m_002#0,"""Ard Banther will fall over after its charging<br>attack if all of its claws have been destroyed."""
+Dead_r01_nat_fan_m_002#0,"""Ard Banther will fall over after its lunging<br>attack if all of its claws have been destroyed."""
```
Comment (SynthSy) [inline Translated/UI/ui_tips_reb.csv]: should be just lunge.

**Review by SynthSy — CHANGES_REQUESTED** [review]

**Pull request merged**